packages/assets-api/package.json

```diff
@@ -32,7 +32,7 @@
   "devDependencies": {
     "@talend/scripts-core": "^11.6.0",
     "@talend/scripts-preset-react-lib": "^9.12.4",
-    "read-pkg-up": "^9.1.0"
+    "read-pkg-up": "^7.0.1"
```

Comment:
Quite the downgrade

Reply:
we already have read-pkg-up in multiple scripts in CJS and I prefere be aligned.
This could be ok here but not in other scripts.
https://github.com/sindresorhus/read-pkg-up/pull/20

It could be upgraded if we move to ESM every where.